model/src/main/java/org/openremote/model/gateway/GatewayCapabilitiesResponseEvent.java

```diff
@@ -10,18 +9,32 @@ public class GatewayCapabilitiesResponseEvent extends SharedEvent {
 
     public static final String TYPE = "gateway-capabilities-response";
     protected final boolean tunnelingSupported;
+    protected final boolean tunnelTimeoutManagementSupported;
+    protected final String version;
 
     @JsonCreator
-    public GatewayCapabilitiesResponseEvent(@JsonProperty("timestamp") Date timestamp, @JsonProperty("tunnelingSupported") boolean tunnelingSupported) {
+    public GatewayCapabilitiesResponseEvent(Date timestamp, boolean tunnelingSupported, String version, boolean tunnelTimeoutManagementSupported) {
         super(timestamp != null ? timestamp.getTime() : new Date().getTime());
         this.tunnelingSupported = tunnelingSupported;
+        this.version = version != null ? version : "0.0.1";
```

Comment:
Don't like this, why fallback to some arbitrary version `0.0.1` leave as null if the gateway doesn't know its' version (i.e. is on a version pre introduction of version in the response)